client/src/Components/RecipeTab/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Recipe } from 'lib/types'
+
+interface Props {
+  recipe: Recipe
+}
+
+export const RecipeTab = ({ recipe }: Props) => {
+  return (
+    <div className='m-10 p-10 bg-purple rounded-xl'>
+      <img className='float-right w-96' src={recipe.image} alt={recipe.name} />
+      <h1 className='text-4xl text-teal font-black'>{recipe.name}</h1>
+      <h3 className='text-2xl text-teal'>Related Characters</h3>
+      <p className='text-white'>{recipe.relatedCharacters.join(', ')}</p>
+      <h2 className='text-2xl text-teal'>Ingredients</h2>
+      <ul>
+        {recipe.ingredients.map((ingredient, index) => (
+          <li key={index} className='text-white'>
+            {ingredient}
+          </li>
+        ))}
```

Comment:
Could we use the ingredient as a key instead of index. It is generally considered an anti pattern using index as a key
![image](https://user-images.githubusercontent.com/23206864/102717233-f182ef00-431b-11eb-90f5-cec18c73e1ca.png)

Reply:
Yep, I was thinking it could be ok since the list is only displayed and never modified on the page - but it makes sense to stick to good practices